CMakeLists.txt

```diff
@@ -9,15 +9,29 @@ cmake_minimum_required(VERSION 3.10)
 # MacOS prior to 10.14 did not support aligned alloc which is used to implement
 # std::unique_ptr in the arm64 C++ standard library. x86_64 builds can override
 # this to 10.13.0 using -DCMAKE_OSX_DEPLOYMENT_TARGET="10.13.0" without issue.
-# This is done in the universal binary building script to build a binary that 
+# This is done in the universal binary building script to build a binary that
 # runs on 10.13 on x86_64 computers, while still containing an arm64 slice.
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.14.0" CACHE STRING "")
 
 set(CMAKE_USER_MAKE_RULES_OVERRIDE "CMake/FlagsOverride.cmake")
 
+# Optionally enable these polcies so older versions of cmake don't break.
+# we only need them for MSVC
+if(POLICY CMP0091)
```

Comment:
setting `CMP0117` to `NEW` should make the `GR` stuff more sane to setup. (or maybe just `cmake_policy(VERSION ${CMAKE_VERSION})` to advance all behavior on cmake/msvc?)